plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -250,6 +250,9 @@ public static function output( $post ) {
 
 					?>
 				</p>
+				<div>
+					<?php do_action( 'woocommerce_order_details_after_payment_info' ); ?>
+				</div>
```

Comment:
A few thoughts here:

* I would recommend just firing the action without adding an extra wrapper to it (since the wrapper would be empty most of the time). The callback can add any wrapper necessary.
* For naming consistency with other action hooks in this meta box, such as `woocommerce_admin_order_data_after_order_details`, what about something like this for the hook name: `woocommerce_admin_order_data_after_payment_info`
* Probably worth adding `$order` as a parameter, like the other actions do.

Reply:
Sounds good, I've made these changes and added a docblock: ff89bfda40a2d0928054e96869b71a8838cb6453

> I would recommend just firing the action without adding an extra wrapper to it (since the wrapper would be empty most of the time). The callback can add any wrapper necessary.

My thinking here was to add a wrapper div (with a class?) to limit the damage that extenders can do - a sandbox of sorts. I.e. make the hook more tailored to specific supported use cases (taken to extreme, could be a `_contextual_notice` filter). Anyway I've removed the div.